### PR TITLE
K8SPXC-438 add possibility to use 32 symbols for cluster name

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-438](https://jira.percona.com/browse/K8SPXC-438)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -117,6 +117,9 @@ pipeline {
         VERSION = "${env.GIT_BRANCH}-${env.GIT_SHORT_COMMIT}"
         CLUSTER_NAME = sh(script: "echo jenkins-pxc-${GIT_SHORT_COMMIT} | tr '[:upper:]' '[:lower:]'", , returnStdout: true).trim()
         AUTHOR_NAME  = sh(script: "echo ${CHANGE_AUTHOR_EMAIL} | awk -F'@' '{print \$1}'", , returnStdout: true).trim()
+
+        //for tests
+        IMAGE_BACKUP = 'perconalab/percona-xtradb-cluster-operator:master-pxc8.0-backup-test'
```
Comment (delgod) [inline Jenkinsfile:122]: should we exclude this line from PR?

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| recreate | passed |
| scaling | passed |
| haproxy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| limits | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| scheduled-backup | failed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| demand-backup-encrypted-with-tls | failed |

File: Jenkinsfile (hunk 1 above)
Comment (hors) [reply Jenkinsfile]: it was for tests

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| scaling | passed |
| haproxy | passed |
| big-data | passed |
| storage | passed |
| init-deploy | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | failed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| self-healing-advanced | passed |
| demand-backup | failed |
| monitoring | passed |
| upgrade-consistency | passed |
| operator-self-healing | passed |
| security-context | failed |
| scheduled-backup | failed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| demand-backup-encrypted-with-tls | failed |

**Review by delgod — APPROVED** [review]

**Pull request merged**